aws/resource_aws_cloudwatch_event_archive.go

```diff
@@ -166,7 +166,8 @@ func buildCreateArchiveInputStruct(d *schema.ResourceData) (*events.CreateArchiv
 	}
 
 	if v, ok := d.GetOk("retention_days"); ok {
-		input.RetentionDays = aws.Int64(v.(int64))
+		retentionInDays := int64(v.(int))
+		input.RetentionDays = aws.Int64(retentionInDays)
```

Comment:
I think we can simplify this to `input.RetentionDays = aws.Int64(v.(int))`

Reply:
That doesn't work: `Cannot use 'v.(int)' (type int) as type int64`. I can however do it one line if that's what you prefer: 

```go
input.RetentionDays = aws.Int64(int64(v.(int)))
```

Reply:
We prefer the inline version

Reply:
You are right, i mistyped :D